.circleci/config.yml

```diff
@@ -16,6 +16,7 @@ jobs:
       - checkout
       - restore_cache:
           key: gemfile-<< parameters.ruby_version >>-{{ checksum "Gemfile" }}
+      - run: gem install bundler:2.6.7
```

Comment:
I think you can just use the default bundler for each ruby...

Reply:
Yes, we don't have a lockfile ... but I think it's okay to remove anyway. It's not like we release this gem very frequently.